docker/geth/entrypoint.sh

```diff
@@ -46,31 +47,43 @@ generate_geneisis()
   mv $tmp $GENISIS_PATH
 }
 
+if [[ -n "$CLEAR_DATA_KEY" && ! -f "$CLEAR_DATA_FILE_PATH" ]]; then
+  echo "Detected change in CLEAR_DATA_KEY. Purging data."
+  rm -rf ${VOLUME_PATH}/*
+  echo "Local data cleared from '${VOLUME_PATH}/*'"
+  touch $CLEAR_DATA_FILE_PATH
+fi
+
 ## One-time configuration to be run only on first startup
 if [[ ! -f $KEYSTORE_PATH && ! -f $SETUP_RUN_PATH ]]; then
-     generate_private_key > $SEALER_PRIVATE_KEY_PATH
-    import_private_key $SEALER_PRIVATE_KEY_PATH > $SEALER_ADDRESS_PATH
+  echo "Generating keys and initializing geth..."
+
+  generate_private_key > $SEALER_PRIVATE_KEY_PATH
+  import_private_key $SEALER_PRIVATE_KEY_PATH > $SEALER_ADDRESS_PATH
 
-    if [ -z "$PRIVATE_KEY" ]; then
-      echo "\nGENERATING PRIVATE KEY!! We most likely don't want to do this.\n"
-      generate_private_key > $PRIVATE_KEY_PATH
-    else
-      echo "Reading private key from env"
-      echo "$PRIVATE_KEY" | sed 's/^0x//' > $PRIVATE_KEY_PATH
-    fi
+  if [ -z "$PRIVATE_KEY" ]; then
+    echo "\nGENERATING PRIVATE KEY!! We most likely don't want to do this.\n"
+    generate_private_key > $PRIVATE_KEY_PATH
+  else
+    echo "Reading private key from env"
+    echo "$PRIVATE_KEY" | sed 's/^0x//' > $PRIVATE_KEY_PATH
+  fi
 
-    import_private_key $PRIVATE_KEY_PATH > $ADDRESS_PATH
+  import_private_key $PRIVATE_KEY_PATH > $ADDRESS_PATH
 
-    generate_geneisis `cat $SEALER_ADDRESS_PATH` `cat $ADDRESS_PATH`
+  generate_geneisis `cat $SEALER_ADDRESS_PATH` `cat $ADDRESS_PATH`
 
-    geth --datadir $VOLUME_PATH --nousb --verbosity 0 init $GENISIS_PATH 2> /dev/null;
-    echo "Ran Setup" > $SETUP_RUN_PATH
+  geth --datadir $VOLUME_PATH --nousb --verbosity 0 init $GENISIS_PATH 2> /dev/null;
+  echo "Ran Setup" > $SETUP_RUN_PATH
 
-    echo "Setup Complete"
-    echo "Sealer Address: 0x`cat $SEALER_PRIVATE_KEY_PATH`"
-    echo "Account Address: 0x`cat $PRIVATE_KEY_PATH`"
+  echo "Setup Complete"
+  echo "Sealer Address: 0x`cat $SEALER_ADDRESS_PATH`"
+  echo "Account Address: 0x`cat $PRIVATE_KEY_PATH`"
```

Comment:
Ah this one too, thanks! 🙂 
```suggestion
  echo "Account Address: 0x`cat $ADDRESS_PATH`"
```